tfjs-converter/src/metadata_test.ts

```diff
@@ -22,7 +22,7 @@ describe('kernel2op metadata file', () => {
   it('has kernel2op.json', () => {
     expect(() => {
       // tslint:disable-next-line:no-require-imports
-      require('../metadata/kernel2op.json');
+      require('tfjs-converter/metadata/kernel2op.json');
```

Comment:
This import breaks with `../` when we use the generated op files (or any generated files for that matter), and I'm not really sure why. Using an absolute import works.